Classes/Services/Transfer/ElasticsearchRepository.php

```diff
@@ -24,6 +24,7 @@
 
 class ElasticsearchRepository implements Repository
 {
+    // obsolete class
```

Comment:
"@Deprecated"?

Reply:
The deprecated classes now marked by using the @deprecated annotation.